packages/@aws-cdk/aws-ec2/lib/volume.ts

```diff
@@ -497,8 +497,10 @@ abstract class VolumeBase extends Resource implements IVolume {
   }
 
   public grantAttachVolumeByResourceTag(grantee: IGrantable, constructs: Construct[], tagKeySuffix?: string): Grant {
```

Comment:
Note: If we're not against breaking this API, then we could remove `tagKeySuffix`. It's probable that no one is using the `Volume` construct yet given that no one seems to have encountered this bug.